### ISSUE TITLE
[Question] V16's "3-3 point Openning" is lasting so long time.

### ISSUE THREAD

**Issue by creeras** [tensorflow/minigo#674]
Through V9 to V15, "3-4" or "4-4" starting was Top-trend. 
And It is a hallmark of other artificial intelligence, including AGZ.

It is so curious that V16's "3-3 point Openning" is lasting so long time.
- Until 18M games, Model No.747

Is it a "overfitting" problem?

**Comment by l1t1** [comment]
pls arrange a match between v15 990 and v16 715

**Comment by atoutw** [comment]
![image](https://user-images.githubusercontent.com/43053753/52176912-0f650a80-27fd-11e9-86d8-6e19af6857f2.png)
v16-805 just took a few seconds from 3-3 to 4-4 by 1080ti (-t11).

**Comment by l1t1** [comment]
https://cloudygo.com/v16-19x19/details/001092-achates v16 1092 still plays 96% 3-3

**Comment by sethtroisi** [comment]
An anonymous user from discord created this interesting graphic

"x axis: model number, y axis: combined policy of 3-3, 3-4, and 4-4. Sampled every 10 models."
![image](https://user-images.githubusercontent.com/10172976/52892352-7a400b00-3146-11e9-8332-032702658453.png)
